docs/faqs.md

```diff
@@ -463,3 +465,11 @@ To have more context about it, please, visit issues #11684 and #11022
 In summary, we do not recommend `full_package_mode` or any other custom package id mode for requirements on CCI, it will break other PRs soon or later.
 Instead, prefer using `shared=True` by default, when needed.
 Also, when having a similar situation, do not hesitate in opening an issue explaining your case, and ask for support from the community.
+
+## Do ConanCenter recipes support editable mode?
+
+No, as they download source code the expectations is that it should not require development - which is the purpose of editable mode. 
+
+Recipes by default should support this however, many use components to model upstream build system configuration files (see above why those are not allowed) as a result there is little support within ConanCenterIndex.
```

Comment:
I don't see here how `using components` is related to `editable mode`, it should be explained more clearly